### PR TITLE
Amended assessment setup to be non-blocking (#1105)

### PR DESCRIPTION
**Pull request by chris-steele**

### CODE REVIEW

**Comment by chris-steele** [comment]
Requires https://github.com/adaptlearning/adapt_framework/pull/1113

**Comment by chris-steele** [comment]
All good from testing this with framework master. If anyone else can have a look over these changes that would be great.

File: js/adapt-assessmentArticleModel.js [hunk 1]
```diff
@@ -460,14 +469,21 @@ define([
...
 
-            for (var i = 0, l = questionComponents.length; i < l; i++) {
-                var question = questionComponents[i];
-                question.reset(assessmentConfig._questions._resetType, true);
+            function step() {
+                for (var j=0, count=Math.min(syncIterations, len-i); j < count; i++, j++) {
+                    var question = qs[i];
+                    question.reset(assessmentConfig._questions._resetType, true);
+                }
+		console.log(i==len?"all reset":"reset question step");
```
Comment (lc-thomasberger) [inline js/adapt-assessmentArticleModel.js]: Can you please remove the console.log.
We might add a develop level log output once the logging system is implemented.

Comment (chris-steele) [inline js/adapt-assessmentArticleModel.js]: @lc-thomasberger done 👍

**Pull request merged**